DOCS-11429-jms-noninclusive-fix-kt (#1543)
mulesoft/docs-connectors: jms/1.1/modules/ROOT/pages/jms-connector-reference.adoc

Used to configure the RedeliveryPolicy#getMaximumRedeliveries().
 0 means no redelivery. -1 means infinite re deliveries are accepted. | 0 |

> | Trusted Packages a| Array of String | Allowed packages of classes to send and receive. For this parameter, ActiveMQ starting versions 5.12.2 and 5.13.0 enforce users to allow packages that can be exchanged using ObjectMessages. |  |

Reviewer: Some ungrammatical text. I suggest something like this but I am not sure if I am interpreting the current text correctly:

... Starting with versions 5.12.2 and 5.13.0, ActiveMQ requires that you set either this parameter or the Trust All Packages parameter to allow the exchange of packages using ObjectMessages.

> | Trusted Packages a| Array of String | Allowed packages of classes to send and receive. Starting with versions 5.12.2 and 5.13.0, ActiveMQ requires that for this parameter you allow packages that can be exchanged using ObjectMessages |  |

Reviewer 2: ```suggestion
| Trusted Packages a| Array of String | Allowed packages of classes to send and receive. Starting with versions 5.12.2 and 5.13.0, ActiveMQ requires you to explicitly allow packages that can be exchanged using ObjectMessages. |  |
```

> | Trust All Packages a| Boolean | Indicates if a class from any package can be sent and received. Enabling this parameter is unsafe because a malicious payload can exploit the host system. For this parameter, ActiveMQ starting versions 5.12.2 and 5.13.0 enforce users to allow packages that can be exchanged using ObjectMessages. | false |

Reviewer: Same text as suggested above

> | Trust All Packages a| Boolean | Indicates if a class from any package can be sent and received. Enabling this parameter is unsafe because a malicious payload can exploit the host system. Starting with versions 5.12.2 and 5.13.0, ActiveMQ requires that for this parameter you allow packages that can be exchanged using ObjectMessages | false |

Reviewer 2: ```suggestion
| Trust All Packages a| Boolean | Indicates if a class from any package can be sent and received. Enabling this parameter is unsafe because a malicious payload can exploit the host system. Starting with versions 5.12.2 and 5.13.0, ActiveMQ requires you to explicitly allow packages that can be exchanged using ObjectMessages. | false |
```
